chainweb.openapi.yaml

```diff
@@ -638,22 +631,20 @@ components:
       description: |
         PoW nonce of the block. This is computed by the miner such that the
         block hash is smaller than the target.
-      type: integer
-      format: int64
-      minimum: 0
-      example: 14452354234648303941
+      type: string
+      pattern: "[0-9]+"
 
     # JSON encoded Block Header
     base64Header:
       title: Base64Url Block Header
       description: Base64Url (without padding) encoded binary block header
       type: string
-      pattern: "[a-zA-Z0-9_-]{424}"
-      minLength: 424
-      maxLength: 424
-      contentEncoding: base64url
+      pattern: "[a-zA-Z0-9_-]+"
+      # minLength: 424
+      # maxLength: 424
       example:
-        $ref: '#/components/examples/base64HeaderPage/items/0'
+        value:
+          $ref: '#/components/examples/base64HeaderPage/value/items/0'
```

Comment:
424 wasn't the length, looks like I forgot to find the real length, I'll fix that.